content/docs/proposals/token-revocation.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Proposal: design for revoking refresh tokens."
```

Comment:
```suggestion
title: "Proposal: design for revoking refresh tokens."
linkTitle: "Proposal: design for revoking refresh tokens"
```

1) Be mindful of punctuation/capitalization on all title fields (Sentence capitalization, no punctuation at the end of titles)
2) Same comment as the above on "Authenticating through...". Remove redundant words from link titles for all in this section.

Reply:
I've gone through and corrected all the title casing throughout, and have removed the redundant words from the linkTitles where they exist.